tests/should-fail/iati-activities/iati-activity/result/reference/err-not-iati-xml-missing-attribute_reference-element-present-no-required-attributes.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+
+<iati-activities version="xx">
+  <iati-activity>
+    <iati-identifier></iati-identifier>
+    <reporting-org type="xx" ref="xx">
+      <narrative>Organisation name</narrative>
+    </reporting-org>
+    <title>
+      <narrative>Xxxxxxx</narrative>
+    </title>
+    <description>
+      <narrative>Xxxxxxx</narrative>
+    </description>
+    <participating-org role="xx"></participating-org>
+    <activity-status code="xx"/>
+    <activity-date type="xx" iso-date="2013-11-27"/>
+    <activity-date type="xx" iso-date="2013-11-27">
+      <narrative>Some stuff ere</narrative>
+    </activity-date>
+    <result type="xx" aggregation-status="0">
+      <title>
+        <narrative>Sample text</narrative>
+      </title>
+      <reference/><!-- FAIL: Optional `reference` element present, but both the mandatory `code` and `vocabulary` attributes are not present. -->
```

Comment:
If they are tested individually, this test file is not required.

(and if they aren't tested individually, this test file doesn't check the things that each is required)

...and there's no manner in XSD 1.0 to say that one of a pair of attributes is required https://stackoverflow.com/a/763108

Reply:
Fixed in 1729f95.